docs/one-click-integrations/github-integration.md

```diff
@@ -139,6 +139,58 @@ The table below lists the data that the GitHub integration collects from your Gi
 
 <sup><span id="deployment-teams">2</span></sup>: Adding a new team or changing the composition of a team on GitHub only affects new data from that moment onwards and doesn't have an immediate impact on the dashboards. Also, deleted teams on GitHub will continue to show in Pulse.
 
+## Which permissions does Pulse need from GitHub? {: id="gh-permissions"}
+
+Pulse requests only the necessary permissions from GitHub to collect your organization changes and deployment data and [keeps your information secure](https://security.codacy.com/). See below the detailed list of permissions.
+
+<table>
+  <colgroup>
+    <col width="20%"/>
+    <col width="20%"/>
+    <col width="60%"/>
+  </colgroup>
+  <thead>
+    <tr>
+      <th>Scope</th>
+      <th>Permissions</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td colspan="3"><strong>Repository permissions:</strong></td>
+    </tr>
+    <tr>
+      <td>Pull requests</td>
+      <td>Read</td>
+      <td>Pulse retrieves pull request information to calculate several of the metrics presented on the dashboards. <a href="#collected-data">See the details here.</a></td>
+    </tr>
+    <tr>
+      <td>Contents</td>
+      <td>Read</td>
+      <td>Pulse retrieves tag information to support semantic versioning. For this, Pulse GitHub App requires read permission on contents of files and directories, as it's the same permission that applies to tags.</td>
+    </tr>
+    <tr>
+      <td>Issues</td>
+      <td>Read</td>
+      <td>Pulse retrieves issue information to get the top-level comments of pull requests.</td>
+    </tr>
+    <tr>
+      <td colspan="3"><strong>Organization permissions:</strong></td>
+    </tr>
+    <tr>
+      <td>Webhooks</td>
+      <td>Read & Write</td>
+      <td>Pulse creates webhooks for organization and repository events (create, pull_request, pull_request_review) to calculate metrics. </td>
```

Comment:
You're absolutely right, since we're using an HTML table the Markdown syntax is no longer interpreted. We can use `<code>` tags instead:

```html
Pulse creates webhooks for organization and repository events (<code>create</code>, <code>pull_request</code>, <code>pull_request_review</code>) to calculate metrics.
```